pkg/pusher/pusher.go

```diff
@@ -219,7 +219,7 @@ func (s *Service) Close() error {
 	// Wait for chunks worker to finish
 	select {
 	case <-s.chunksWorkerQuitC:
-	case <-time.After(3 * time.Second):
+	case <-time.After(6 * time.Second):
```

Comment:
you could theoretically return an error here, and then it would be printed in the shutdown sequence. this should be done on other components too (but as a separate issue)

Reply:
This point should not be reached due to other change, with lower timeout.

However, it might be useful to have that other change in another small loop, and print some message every second (or so) while waiting to stop worker ?

Reply:
it should not be reached, but if it does - we should get some sort of signalling. you can have a look at other services that shut down and have this sort of shutdown sequence, they usually print a warning when the timeout is reached without clean shutdown. this should ideally be replaced by using the errors returned in the shutdown sequence in `node.go` (at least i am at this opinion)

Reply:
about printing the message at intervals :+1: but janos was at an opposition to this

Reply:
There is one such log line next to other changed timeout, and I have added `s.logger.Info("pusher shutting down")` line in the beginning of `Close()` function.
This makes it same as for `pullsync` package.

Reply:
On another point, I have checked how it looks when printing log every second or so yesterday, and did not like how it looks (that is personal opinion). Unless we can change existing line with some progress (while possible, we should not bother here), we probably should not print on interval until closed.
I think that line where notifying user that operation has started is good enough.

Reply:
sounds good :+1: